charts/fluent-bit.yaml

```diff
@@ -0,0 +1,126 @@
+---
+loki:
+  serviceName: loki  # Defaults to "${RELEASE}-loki" if not set
+  servicePort: 3100
+  serviceScheme: http
+  servicePath: /api/prom/push
+  # user: user
+  # password: pass
+config:
+  port: 2020
+  tenantID: '""'
+  batchWait: 1
+  batchSize: 10240
+  loglevel: warn
+  lineFormat: json
+  k8sLoggingParser: "Off"
+  removeKeys:
+    - kubernetes
+    - stream
+  autoKubernetesLabels: false
+  labels: '{job="fluent-bit"}'
+  labelMap:
+    kubernetes:
+      namespace_name: namespace
+      labels:
+        app: app
+        release: release
+      host: node
+      container_name: container
+      pod_name: instance
+    stream: stream
```

Comment:
Since you do not use the config from the chart maybe it makes sense to comment this section and add a comment to say that we do not use this one from the chart as we cannot provide multiple inputs